src/main/scala/beam/agentsim/agents/ridehail/RideHailAgent.scala

```diff
@@ -167,6 +181,10 @@ class RideHailAgent(
       stay()
 
   }
+  onTransition {
```

Comment:
Was this broken before? I notice some stashs, but never unstashing. I guess I just want to make sure we are careful of this as it is a definite behavior change...probably for the good though - as the old behavior was odd...